### PR TITLE
Fix bag branch

### LINKED ISSUE #1: LCD init lib
Include in menu Liquid Crystal Base library

### PR DESCRIPTION
**Pull request by IgorIoT13**

### CODE REVIEW

File: Dragee.ino
Comment (VladVanyuk) [inline Dragee.ino:1]: Move to another file what is not necessary.

File: Enc.h
Comment (VladVanyuk) [inline Enc.h:1]: Deleted?

File: pins.h
Comment (VladVanyuk) [inline pins.h:1]: Delete RelayPIN  and other TESTs values

File: relay.h
Comment (VladVanyuk) [inline relay.h:1]: bool* pidState ;
Whats your point? That's make no sense

File: secrets.h
Comment (VladVanyuk) [inline secrets.h:1]: Make dynamic and changeable wifi_mode

File: serverHost.h
Comment (VladVanyuk) [inline serverHost.h:1]: Why includes are out of range of #ifdef ?

File: settings.h
Comment (VladVanyuk) [inline settings.h:1]: Add 
min_heat_temp = 30;

File: countUpTimer.h
Comment (VladVanyuk) [inline countUpTimer.h:1]: Are you sure it will not be necessary in future?

**Review by VladVanyuk — CHANGES_REQUESTED** [review]
Make changes described in comment blocks. @IgorIoT13

File: secrets.h
Comment (IgorIoT13) [reply secrets.h:1]: Done

File: settings.h
Comment (IgorIoT13) [reply settings.h:1]: Done

File: countUpTimer.h
Comment (IgorIoT13) [reply countUpTimer.h:1]: Yes, because it is already integrated into the class

File: Enc.h
Comment (IgorIoT13) [reply Enc.h:1]: Don't understend

File: pins.h
Comment (IgorIoT13) [reply pins.h:1]: Done

File: serverHost.h
Comment (IgorIoT13) [reply serverHost.h:1]: Done

File: Dragee.ino
Comment (IgorIoT13) [reply Dragee.ino:1]: Done?

File: Enc.h
Comment (IgorIoT13) [reply Enc.h:1]: Done

**Review by VladVanyuk — APPROVED** [review]

**Pull request merged**